app/views/placements/support/_primary_navigation.html.erb

```diff
@@ -0,0 +1,10 @@
+<%# locals: (current_navigation: nil) -%>
+
+<%= content_for(:primary_navigation_content) do %>
+  <%= render PrimaryNavigationComponent.new do |component| %>
+    <% component.with_navigation_item t(".organisations"),
+                                      support_organisations_path,
+                                      current: current_navigation == :organisations %>
```

Comment:
There's a potential way to condense this logic.

```suggestion
                                      current: controller_name == "placements/support/organisations" %>
```

```suggestion
                                      current: controller_name.include?("support/organisations") %>
```

Reply:
And if you also need to support the schools and providers controllers:

```suggestion
                                      current: %w[placements/support/organisations placements/support/providers placements/support/schools].include?(controller_name) %>
```

Reply:
Not a blocker.

Reply:
I feel like this loses the explicit, easy-to-read nature of what was there before.